wReleaseNotes/2024-06_ReleaseNotes.md

```diff
@@ -3,19 +3,48 @@ title: Release Notes
 filename: "2024-06_ReleaseNotes.md"
 layout: custom
 keywords: [change log, updates, versions, history, what's new]
-headings: ["June 2024", "Documentation Updates"]
-links: []
+headings: ["June 2024", "Excel Addin v2.5.1.1", "Portal Site", "Documentation Updates"]
+links: ["/wAbout/SingleUser.html", "/wAbout/SharedComputer.html", "/wPortal/Roles.html", "/wPortal/FolderPermissions.html"]
 image_dir: ""
 images: []
 description: Release notes for this month's updates
 ---
 * * *
 
-## May 2024
+## June 2024
+
+### Excel Addin v2.5.1.1
+
+_Released 6/4/24_
+
+- 🐞 Wrapped decimal conversion in try/catch where excel versions were being read in addin manager. This fixes a bug where setting the region on your machine causes the addin manager to not open.
+
+- ✅ The main settings and login information are moved from the AppData local to AppData roaming so that when a user is in a shared environment with multiple possible windows instances they could log into, their Interject login and settings can roam between them. This will have little to no effect on single user computers as the roaming folder always exists and persists between logins.
+
+- ✅ Adds a hook to sanitize incoming logs for potentially sensitive data that is included in parameters in error logs or database errors controlled by third party, not reports. Currently only looking for SSNs.
```

Comment:
this sentence is really hard to understand. kind of needs a total revamp. maybe "adds a hook to sanitize incoming parameters in error logs with potentially sensitive data, or in database errors controlled by third parties. this doesn't include reports, and currently only looks for social security numbers."